packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -917,4 +917,74 @@
       <protected>false</protected>
       <shortDescription>Show less</shortDescription>
     </labels>
+    <labels>
```

Comment:
@acarpentier01 your help will be much appreciated here, let me know if there are any improvements you think I can add to these labels.